website/pages/docs/reference/cli/cloudquery_validate-config.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
You also need to update https://github.com/cloudquery/cloudquery/blob/f509e8b1c961427979cdb3a525e7bc88bdaea308/cli/cmd/doc_test.go#L11